test/run-openshift-remote-cluster

```diff
@@ -11,18 +11,20 @@ THISDIR=$(dirname ${BASH_SOURCE[0]})
 source "${THISDIR}/test-lib.sh"
 source "${THISDIR}/test-lib-openshift.sh"
 source "${THISDIR}/test-lib-python.sh"
+source ${THISDIR}/test-lib-remote-openshift.sh
 
 set -eo nounset
+ct_os_set_ocp4
+
 
 trap ct_os_cleanup EXIT SIGINT
 
 ct_os_check_compulsory_vars
 
 oc status || false "It looks like oc is not properly logged in."
 
-export CT_SKIP_NEW_PROJECT=true
-export CT_SKIP_UPLOAD_IMAGE=true
-export CT_NAMESPACE=openshift
+# For testing on OpenShift 4 we use internal registry
+export CT_EXTERNAL_REGISTRY=true
```

Comment:
To me, it looks like: the comment says we use the **internal** registry and the line below it enables **external** registry. Am I wrong? This is just a nitpick.

Reply:
internal registry means docker-registry.upshift.redhat.com and CT_EXTERNAL_REGISTRY mean export to this internal registry. I will update it be next commit.